blog/2020-05-16-web-support.md

```diff
@@ -0,0 +1,109 @@
+        <Stack.Screen name="Profile" component={Profile} />
+        <Stack.Screen name="Settings" component={Settings} />
+      </Stack.Navigator>
+    </NavigationContainer>
+  );
+}
+```
+
+We have a [playground](https://reactnavigation.org/docs/configuring-links/#playground) where you can try custom configurations and see how it's parsed. Give it a try, and we hope it'll make it easier for you to configure links in your apps.
+
+There's a new [`<Link />` component](https://reactnavigation.org/docs/link) which lets you use URLs for navigation in the app.
+
+Example:
+
+```js
+<Link to="/profile/jane">Go to Jane's profile</Link>
+```
+
+When you use this component, it renders anchor tags on the web. This means that users can use the same patterns that they are used to on the web, such as "Open in new tab", "Copy link address" etc.
+
+Special thanks to [Michał](https://github.com/osdnk) and [Wojciech](https://github.com/WoLewicki) for their extensive work on URL integration.
+
+## Improvements to the navigators
+
+URL integration isn't enough to have proper web support. The navigators also need to feel at home on the web. We have made several improvements to the built-in navigators so that they behave as you expect on the web.
+
+### Anchor tags in tabs and drawer
+
+The first change is using anchor tags. When you use built-in navigators such as drawer navigator and tab navigator, they render anchor tags for the drawer and tab items respectively when you have linking configured. This means that they behave the same as normal links on the web.
+
+<img src="/assets/blog/web-support/link-right-click.png" height="427"/>
+
+### No gestures and animations
+
+Another change is the removal of gestures on the web. Gestures are not commonly used on the web because they conflict with a lot of browser functionality and system gestures. For example, in Safari, you can swipe to go back to the previous page. Android has a system gesture to go back to previous pages too. Because of this, we've removed gestures on the web.
+
+We've also disabled animations on the web by default since they are not commonly used and can be jarring, especially on larger screens. The animations in React Native Web also run entirely in JS, and tend not to be very performant on the web. So we've decided to disable them by default to better overall experience.
+
+### Hide address bar on scroll
+
+Phones have small screens, so it's important to maximize the use of available screen size. One of the ways mobile browsers achieve it is by hiding and showing the address bar when scrolling to give more vertical space to the content.
+
+It's especially important in case of the [stack navigator](https://reactnavigation.org/docs/stack-navigator) because not only we have the browser's address bar, but also the header at the top which is taking vertical space. Now we'll automatically adjust the styles of the stack navigator to get this behavior you having to write any special code.
```

Comment:
```suggestion
It's especially important in case of the [stack navigator](https://reactnavigation.org/docs/stack-navigator) because not only we have the browser's address bar, but also the header at the top which is taking vertical space. Now we'll automatically adjust the styles of the stack navigator to get this behavior without you having to write any special code.
```